specs/state-proof.md

```diff
@@ -92,25 +98,33 @@ to not be in the table.
 
 ### Tx Access List Account
 - 9.0. `field_tag` and `storage_key` are 0
-- 9.1. `state root` is the same
-- 9.2. First access for a set of all keys are 0 if `READ`
-
+- 9.1. `value` is boolean
+- 9.2. `initial_value` is false
+- 9.3. `state root` is the same
+- 9.4. First access for a set of all keys are 0 if `READ`
+- 9.5. `value` column at previous rotation equals `value_prev` at current rotation
 
 ### Tx Access List Account Storage
 - 10.0. `field_tag` is 0
-- 10.1. `state root` is the same
-- 10.2. First access for a set of all keys are 0 if `READ`
+- 10.1. `value` is boolean
+- 10.2. `initial_value` is false
+- 10.3. `state root` is the same
+- 10.4. First access for a set of all keys are 0 if `READ`
+- 10.5. `value` column at previous rotation equals `value_prev` at current rotation
 
 ### Tx Log
 - 11.0. `is_write` is 1
-- 11.1. `state root` is the same
+- 11.1. `initial_value` is 0
+- 11.2. `state root` is the same
+- 11.3. `value_prev` equals `initial_value`
 
 ### Tx Receipt
 - 12.0. `address` and `storage_key` are 0
 - 12.1. `field_tag` is boolean (according to EIP-658)
 - 12.2. `tx_id` increases by 1 and `value` increases as well if `tx_id` changes 
 - 12.3. `tx_id` is 1 if it's the first row and `tx_id` is in 11 bits range
 - 12.4. `state root` is the same
+- 12.5. `value_prev_column` is 0
```

Comment:
This should be `value_prev` (without `column`) and `initial_value` is 0 as well.

Reply:
Would it make sense to rename value_prev and value_prev_column (edit: in the code)? I find these a bit confusing.

`value_prev: WordLoHi<Expression<F>>, // meta.query(value, Rotation::prev())` could be renamed `value_rot_prev` for instance
`pub value_prev_column: WordLoHi<Expression<F>>, // meta.query(prev_value, Rotation::cur())` be renamed `value_prev`

Reply:
Fixed in [12.5 both value_prev and initial_value are 0](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/520/commits/35875e8e1b04b049f1c7788ada940fb1ef59d051)